src/task/CreatePeerConnectionTask.ts

```diff
@@ -187,7 +194,26 @@ export default class CreatePeerConnectionTask extends BaseTask implements Remova
       width = cap.width as number;
       height = cap.height as number;
     }
-    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId);
+    const externalUserId = this.context.realtimeController.realtimeExternalUserIdFromAttendeeId(
+      attendeeId
+    );
+    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId, externalUserId);
+    if (!externalUserId) {
+      const handler = (
+        presentAttendeeId: string,
+        present: boolean,
+        externalUserId: string
+      ): void => {
+        if (attendeeId === presentAttendeeId && present && externalUserId !== null) {
+          tile.bindVideoStream(attendeeId, false, stream, width, height, streamId, externalUserId);
+          this.context.realtimeController.realtimeUnsubscribeToAttendeeIdPresence(handler);
+          this.presenceHandlerSet.delete(handler);
```

Comment:
This implementation assumes that the presence handler is going to be called. What happens to the attendee presence event does not arrive (e.g. connection is lost)? How will this handler be cleaned up in that case?

I would suggest setting a timer that will automatically delete the handler if it is not called after a certain amount of time (say 10 seconds).

Reply:
Have added a TimeoutScheduler to removeHandlers after every 10 seconds.